app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
Go with `haml`. It is more readable, IMO

[source](https://github.com/k0kubun/hamlit)

Reply:
[html2haml](https://github.com/haml/html2haml) can save you some time with conversion